plugin-server/package.json

```diff
@@ -28,8 +30,7 @@
         "services:stop": "cd .. && docker compose -f docker-compose.dev.yml down",
         "services:clean": "cd .. && docker compose -f docker-compose.dev.yml rm -v",
         "services": "pnpm services:stop && pnpm services:clean && pnpm services:start",
-        "build:cyclotron": "pnpm --filter=@posthog/cyclotron package",
-        "pnpm:devPreinstall": "pnpm run build:cyclotron",
+        "build:cyclotron": "pnpm --filter=@posthog/plugin-server build:cyclotron",
```

Comment:
Isn't this an endless loop? 🤔

Reply:
ah whoops that was me playing and lazily commiting

Reply:
good catch 😅